Dockerfile

```diff
@@ -8,11 +8,8 @@ RUN yum --assumeyes install \
     bash-completion \
     findutils \
     fzf \
-    git \
-    golang \
     jq \
     krb5-workstation \
-    make \
```

Comment:
I sometimes use these, but if you see this an not used, we can remove and review the list later

Reply:
Hrm, if they're useful, we don't need to necessarily remove them. Do you use them frequently?  If not, since the container is running as root, is it OK to just install when needed?

I'm just trying to clean up a bit, but if these are frequent and helpful, I'm happy to leave them in.

Reply:
Nah, I'll see how much I use them and worst case I can install